src/main/java/frc/robot/Constants.java

```diff
@@ -87,7 +105,7 @@ public static final class RobotConstants {
     public static final int kRearLeftTurningCanId = 8;
 
     // Used to declare Navx as upside down
-    public static final boolean kGyroReversed = false;
+    public static final boolean kGyroReversed = true;
```

Comment:
Leave a note here about the behavior of the robot if this is reversed so future us can find connect the setting to the behavior again if needed.